docs/source/guide/predictions.md

```diff
@@ -90,6 +90,376 @@ The prediction score applies to the entire prediction.
   
 
 In the Label Studio UI, the imported prediction for this task looks like the following: 
-<center><img src="../images/predictions_loaded.png" style="width: 100%; max-width: 700px"></center>
+<center><img src="../images/predictions_loaded.png" alt="screenshot of the Label Studio UI showing an image of airplanes with bounding boxes covering each airplane." style="width: 100%; max-width: 700px"></center>
+
+<!-- md image_units.md -->
+
+## Import pre-annotations for text 
+
+In this example, import pre-annotations for text using the [named entity recognition template](/templates/named_entity.html):
+```xml
+<View>
+  <Labels name="label" toName="text">
+    <Label value="Person"></Label>
+    <Label value="Organization"></Label>
+    <Label value="Fact"></Label>
+    <Label value="Money"></Label>
+    <Label value="Date"></Label>
+    <Label value="Time"></Label>
+    <Label value="Ordinal"></Label>
+    <Label value="Percent"></Label>
+    <Label value="Product"></Label>
+    <Label value="Language"></Label>
+    <Label value="Location"></Label>
+  </Labels>
+  <Text name="text" value="$text"></Text>
+</View>
+```
+
+This example JSON file contains two tasks, each with two sets of pre-annotations from different models. The first task also contains prediction scores for each NER span. After you set up an example project, you can import these tasks into Label Studio after saving it as a file, such as `example_preannotated_ner_tasks.json`.
+
+```json
+[
+  {
+    "data": {
+      "text": "All that changed when he was 27 and he came to Jerusalem. It was the weekend of both Easter and Passover, and the city was flooded with tourists."
+    },
+    "predictions": [
+      {
+        "model_version": "one",
+        "result": [
+          {
+            "from_name": "label",
+            "to_name": "text",
+            "type": "labels",
+            "value": {
+              "start": 29,
```

Comment:
What if we reduce this subdict to one line? 

```
"value": {
              "start": 29,
              "end": 31,
              "score": 0.70,
              "text": "27",
              "labels": [
                "Date"
              ]
            }
```

=>

```
"value": { "start": 29, "end": 31, "score": 0.70, "text": "27", "labels": ["Date"] }
```

Also we can do the same for 

```
            "from_name": "label",
            "to_name": "text",
            "type": "labels",
```

=>

```
            "from_name": "label", "to_name": "text", "type": "labels",
```

I'm worried about the size of this example: it's huge...

Reply:
I could put it behind a read more expandable thing so that it doesn't look so scary on the page. I think the more readable JSON is helpful to have laid out.

Reply:
Please check the latest, I added an expander